core/root/config/templates/cluster.yaml

```diff
@@ -269,7 +269,13 @@ worker:
 #  # Required if there are 2 or more API endpoints defined in `apiEndpoints`
 #  # Can be overriden per node pool by specifying `worker.nodePools[].apiEndpointName`
 #  apiEndpointName: versionedPublic
-#
+#  # Used to globally specify nodePool rolling strategy across availability zones: 'Sequential' or 'Parallel'
```

Comment:
Can you safely remove the part `across availability zones` from the explanation?
I'm saying it because it seems like we don't actually take AZs of node pools into account, but rather the one of expected use-cases is to roll parallel per AZ.